apps/nextjs-website/react-components/types/VideoImage/VideoImage.types.ts

```diff
@@ -27,8 +28,9 @@ export interface VideoImageProps
 
 export interface VideoTextProps {
   readonly title?: string;
-  readonly subtitle?: string;
+  readonly subtitle?: string | JSX.Element;
```

Comment:
```suggestion
  readonly subtitle?: JSX.Element;
```
Only allow for rendered markdown to be passed